README.md

```diff
@@ -753,7 +753,11 @@ public class MyComponentDialog { /* ... */ }
 
 #### Fields inheritance and ways to cancel it
 
-Same as dialog tabs, dialog fields are "inherited" across the Java classes. However, there is the possibility to "cancel" a superclass-bound field from rendering in a current dialog or a narrower container. Add `@IgnoreFields` annotation to the current class:
+Same as dialog tabs, dialog fields are "inherited" across Java classes. Dialog fields from a superclass and from a child class are always considered two different fields, event if sharing same name. If a superclass contains a field like `@DialogField @TextField private String text;`, and its child class contains a `@DialogField @TextField private String text;` of its own, *both*  will be rendered in a dialog for the child class.
+
+Note that the presence of two or more same-named fields in a single "class with its ancestors - to - dialog" context will not break the rendering, but may cause trouble as to saving authored data. That is why `InvalidFieldContainerException` is thrown in such a case.  
```

Comment:
Also, not exactly true, the exception is thrown only in parent's filed has a higher ranking than child's field